index.bs

```diff
@@ -868,7 +868,7 @@ To support obtaining assertions via {{CredentialsContainer/get()|navigator.crede
 </xmp>
 
 
-### Create a new credential - PublicKeyCredential's `[[Create]](origin, options, sameOriginWithAncestors)` method ### {#createCredential}
+### Create a New Credential - PublicKeyCredential's `[[Create]](origin, options, sameOriginWithAncestors)` Method ### {#createCredential}
```

Comment:
POI: I'm not entirely sure we want to capitalize "Method" here and in similar headings, but I think so.